Pd 83 scale 22.12 update i scsi article content with bluefin changes (#1720)
truenas/documentation: content/SCALE/SCALETutorials/Shares/iSCSI/AddingISCSIShares.md

   a. Click **Configure** on the main **Block (iSCSI) Share Targets** widget.
      The **Target Global Configuration** tab screen opens.

      ![iSCSIManualTargetGlobalConfig](/images/SCALE/22.12/iSCSIManualTargetGlobalConfig.png "iSCSI Target Global Configuration")

   b. Enter a name using lowercase alphanumeric characters plus dot (.), dash (-), and colon (:) in **Base Name**. 
      Use the iqn.format for the name. See the "Constructing iSCSI names using the iqn.format" section of [RFC3721](https://tools.ietf.org/html/rfc3721.html).

   c. Enter the host names or IP address of the ISNS servers to register with the iSCSI targets and portals of the system. Separate entries by pressing <kbd>Enter</kbd>.

   d. The value entered in **Pool Available Space Threshold** generates an alert when the pool has this percentage of space remaining. This is typically configured at the pool level when using zvols or at the extent level for both file and device based extents. 

>    e. Enter the **iSCSI listen port**. Add the TCP port used to access the iSCSI target. Default is `3260`.

Reviewer: ```suggestion
   e. Enter the **iSCSI listen port**.  Add the TCP port used to access the iSCSI target. The default is `3260`.
```

   d. (Optional) Based on your **Discovery Authentication Method**, select a group in **Discovery Authentication Group**.

>    e. Click **Add** to select an **IP Address** to be listened on by the portal from the dropdown list. **0.0.0.0** listens on all IPv4 addresses and **::** listens on all IPv6 addresses. 

Reviewer: ```suggestion
   e. Click **Add** to select an **IP Address** the portal listens on from the dropdown list. **0.0.0.0** listens on all IPv4 addresses and **::** listens on all IPv6 addresses. 
```

   f. Click **Save**.

3. Add initiators groups to create authorized access client groups. Click on the **Initiators Groups** tab to open the screen.

   a. Click **Add** at the top right of the screen to open the **SHARING > ISCSI > INITIATORS > Add** screen.
      
   ![iSCSIManualAddInitiators](/images/SCALE/22.12/iSCSIManualAddInitiators.png "iSCSI Initiators Groups")

   b. Select **Allow All Initiators** or configure your own allowed initiators.

      Enter the [iSCSI Qualified Name (IQN)](https://tools.ietf.org/html/rfc3720#section-3.2.6) in **Allowed Initiators (IQN)** and click **+** to add it to the list. Example: *iqn.1994-09.org.freebsd:freenas.local*.

   c. Click **Save**.

4. Add network authorized access. Click on the **Authorized Access** tab to open the screen.
   

>    a. Click **Add** at the top right of the screen. The **Add Authorized Access** screen opens.

Reviewer: ```suggestion
   a. Click **Add** at the top right of the screen to open the **Add Authorized Access** screen.
```

   f. Click **Save**.

5. Create storage resources. Click **Targets** tab. 
   
   ![iSCSIManualAddTargets](/images/SCALE/22.12/iSCSIManualAddTargets.png "iSCSI Add Targets")
   

>    a. Click **Add** at the top right of the screen. The **Add iSCSI Target** screen opens.

Reviewer: ```suggestion
   a. Click **Add** at the top right of the screen to open the **Add iSCSI Target** screen.
```

   b. Enter a name in **Target Name**. Use lowercase alphanumeric characters plus dot (.), dash (-), and colon (:) in the iqn.format. 
      See the "Constructing iSCSI names using the iqn.format" section of [RFC3721](https://tools.ietf.org/html/rfc3721.html).

   c. (Optional) Enter a user-friendly name in **Target Alias**.

   d. Click **Add** next to **Authorized Networks** to enter IP address information.

   e. Click **Add** under **iSCSI Group** to display the group settings.

   f. Select the group ID from the **Portal Group ID** dropdown.

   g. (Optional) Select the group ID in **Initiator Group ID** or leave it set to **None**.

   h. (Optional) Select the **Authentication Method** from the dropdown list of options.

   i. (Optional) Select the **Authentication Group Number** from the dropdown list. This value represents the number of existing authorized accesses.
   
   j. Click **Save**.

6. Add new share storage units (extents). Click the **Extents** tab.
   
   ![iSCSIManualAddExtentDevice](/images/SCALE/22.12/iSCSIManualAddExtentDevice.png "iSCSI Extents Add Device")
 

>    a. Click **Add** at the top right of the screen. The **Add Extent** screen opens.

Reviewer: ```suggestion
   a. Click **Add** at the top right of the screen to open the **Add Extent** screen.
```

   b. Enter a name for the extent. If the extent size is not **0**, it cannot be an existing file within the pool or dataset.

   c. Leave **Enabled** selected.

   d. In the **Compatibility** section, the **Enable TPC** checkbox is selected by default. This allows an initiator to bypass normal access control and access any scannable target.

>    e. The **Xen initiator compat mode** checkbox is not selected by default. Select when using Xen as the iSCSI initiator.

Reviewer: ```suggestion
   e. **Xen initiator compat mode** is disabled by default. Select when using Xen as the iSCSI initiator.
```

>    f. The **LUN RPM** setting should not be changed when using Windows as the initiator. Only needs to be changed in large environments where the number of systems using a specific RPM is needed for accurate reporting statistics. 

Reviewer: ```suggestion
   f. Do not change **LUN RPM** when using Windows as the initiator. Only change **LUN RPM**  in environments where you need accurate reporting statistics for devices that use a specific RPM. 
```

>    f. The **LUN RPM** setting should not be changed when using Windows as the initiator. Only needs to be changed in large environments where the number of systems using a specific RPM is needed for accurate reporting statistics. 

Reviewer: The UI tooltip is inaccurate.

>    g. The **Read-only** checkbox is not selected by default. Select to prevent the initiator from initializing this LUN.

Reviewer: ```suggestion
   g. **Read-only** is disabled by default. Select to prevent the initiator from initializing this LUN.
```

      **Device** provides virtual storage access to zvols, zvol snapshots, or physical devices. 
      **File** provides virtual storage access to a single file.

   i. (Optional) Select the option from the **Device** dropdown. This field only displays when **Extent Type** is set to **Device**.
      Select the path when **Extent Type** is set to **File**. Browse to the location. 
      Create a new file by browsing to a dataset and appending /\{filename.ext\} to the path. Enter the size in **Filesize**.

      ![iSCSIManualAddExtentFile](/images/SCALE/22.12/iSCSIManualAddExtentFile.png "iSCSI Extents Add File Name") 

   j. Select the **Logical Block Size** from the dropdown list. Leave at the default of 512 unless the initiator requires a different block size.

   k. Select **Disable Physical Block Size Reporting** if the initiator does not support physical block size values over 4K (MS SQL).

   l. Click **Save**.
   
7. Add associated storage resources. Click **Associated Targets** tab.
      
      ![iSCSIManualAddAssocAuthTargets](/images/SCALE/22.12/iSCSIManualAddAssocAuthTargets.png "iSCSI Add Associated Targets")

>    a. Click **Add** at the top right of the screen. The **Add Associated Target** screen opens.

Reviewer: ```suggestion
   a. Click **Add** at the top right of the screen to open the **Add Associated Target** screen.
```

   b. Select the target from the **Target** dropdown list.

>    c. Enter a value 0 and 1023 for the **LUN ID**. Some initiators expect a value below 256. Leave blank to automatically assign the next available ID.

Reviewer: ```suggestion
   c. Select the value or enter a value between 0 and 1023. Some initiators expect a value below 256. Leave this **LUN ID** blank to automatically assign the next available ID.
```

   d. Select an existing extent from the **Extent** dropdown.

   e. Click **Save**
{{< /expand >}}

## Creating a Quick iSCSI Target
TrueNAS SCALE allows users to add iSCSI targets without having to set up another share.
{{< expand "Click here for more information" "v" >}}
Go to **Shares** and click the **Block (iSCSI) Shares Targets** widget.

>    a. Click **Add** at the top right of the screen. The **Add iSCSI Target** screen opens.

Reviewer: ```suggestion
   a. Click **Add** at the top right of the screen to open the **Add iSCSI Target** screen.
```
